sql/connect/common/src/main/protobuf/spark/connect/pipelines.proto

```diff
@@ -146,11 +141,26 @@ message PipelineCommand {
 message PipelineCommandResult {
   oneof result_type {
     CreateDataflowGraphResult create_dataflow_graph_result = 1;
+    DefineDatasetResult define_dataset_result = 2;
+    DefineFlowResult define_flow_result = 3;
+  }
+  message CatalogIdentifier {
```

Comment:
@gengliangwang @cloud-fan – thoughts on whether `CatalogIdentifier` is the right name and whether this is the right location for this message? Since this is a type that might end up useful elsewhere as well, I wonder if it would make more sense as a top-level message inside base.proto or catalog.proto.